app/jinja_filters.py

```diff
@@ -93,20 +98,18 @@ def format_unit_input_label(unit: str, unit_length: UnitLengthType = "short") ->
     :param (str) unit_length length of unit text, can be one of short/long/narrow
     """
     unit_label: str
+    # mass-metric-ton no longer supported for en_GB and related locales, but still present in business schema and allowed in validator,
+    # until removed from schema we substitute mass-tonne for mass-metric-ton before format unit
+    unit = "mass-tonne" if unit == "mass-metric-ton" else unit
```

Comment:
an advantage to this refactor is that you don't need this line anymore, because `format_unit` is already doing the replacement of `mass-metric-ton` with `mass-tonne`

Reply:
Oh, yeah, sorry distracted.